src/main/kotlin/eu/kevin/api/Endpoint.kt

```diff
@@ -6,5 +6,6 @@ internal object Endpoint {
 
     object Path {
         const val INITIATE_PAYMENT = "/pis/payment"
+        fun initiatePaymentRefund(paymentId: String) = "/pis/payment/$paymentId/refunds"
```

Comment:
uhmm.. I don't think this is a right way to format string. Better use string template
`"/pis/payment/%s/refunds"` and then format it with String.format(x, paymentId)

Reply:
I guess it would not be very convenient, since for a particular endpoint, we would not know if we need to apply string formatting, also which and how many parameters to provide. Also makes refactoring harder.

Reply:
There are tests for that. I don't see how refactoring becomes harder.

Reply:
Why write tests if compiler could check this thing compile time? If an additional path parameter would be added to the path, we would have to update the function call to compile, where as with String.format(), we may forget to update it

Reply:
@tkopalinskas what do you think?

Reply:
I agree with Edgar. Even tho it is more convenient to use it as it is now, consistency is key, and test coverage would provide safety net.

Reply:
@grantas33 power of 2 is greater than power of 1 😄